daml-script/daml-v2/daml-script-index-template.rst

```diff
@@ -0,0 +1,11 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _daml-script-api-docs:
+
+Daml Script Library
```

Comment:
Maybe it would be good to write somewhere this is a developement version of a new version of script.

Reply:
Ah yes, I'll update the templates